docs/resources/samples/combine-worksheets-into-single-workbook.md

```diff
@@ -96,7 +96,7 @@ interface WorksheetData {
     * **File**: Your file
     * **Script**: Add worksheets
     * **workbookName**: *Name* (dynamic content from **List files in folder**)
-    * **worksheetInformation**: *result* (dynamic content from **Run script**)
+    * **worksheetInformation** (after pressing the **Switch to input entire array** button - see the following note): *result* (dynamic content from **Run script**)
```

Comment:
I think the guidance is to say "select" or "choose" instead of "press" or "click"
```suggestion
    * **worksheetInformation** (after selecting the **Switch to input entire array** button, see the note following the next image): *result* (dynamic content from **Run script**)
```